core/src/main/java/com/scalar/db/config/DatabaseConfig.java

```diff
@@ -62,6 +64,7 @@
   public static final String TRANSACTION_MANAGER = PREFIX + "transaction_manager";
   public static final String TABLE_METADATA_CACHE_EXPIRATION_TIME_SECS =
       PREFIX + "table_metadata.cache_expiration_time_secs";
+  public static final String NEED_OPERATION_COPY = PREFIX + "need_operation_copy";
```

Comment:
I added a new configuration `scalar.db.need_operation_copy` where the operation copy in `DistributedStorage` is disabled when it's set to `false`.
Honestly, I'm wondering if it's a good configuration name.